ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -637,3 +639,67 @@ def test_paddle_take_along_axis(
         indices=indices,
         axis=axis,
     )
+
+
+@st.composite
+def put_along_axis_helper(draw):
+    input_dtype, x, axis, shape = draw(
+        helpers.dtype_values_axis(
+            available_dtypes=helpers.get_dtypes("valid"),
+            min_num_dims=2,
+            min_dim_size=2,
+            valid_axis=True,
+            force_int_axis=True,
+            ret_shape=True,
+            min_axis=0,
+            max_axis=1,
+        )
+    )
+
+    if axis < 0:
+        axis = 0
```

Comment:
I think we can remove this since you have set `min_axis=0` above.

Reply:
It was drawing negative values during some test runs

Reply:
Could you add a ToDo comment there to keep track of the problem then? Something like "`dtype_values_axis` currently returns negative axis even for min_axis=0. Remove this condition when the issue is fixed."